packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -0,0 +1,319 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import Button from 'terra-button';
+import ContentContainer from 'terra-content-container';
+import IconClose from 'terra-icon/lib/icon/IconClose';
+import IconLeft from 'terra-icon/lib/icon/IconLeft';
+import MenuDivider from './_UtilityMenuDivider';
+import Utils from '../_Utils';
+import MenuItem from './_UtilityMenuItem';
+import styles from './_UtilityMenu.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The initial selected key. Used as the top level menu page.
+   */
+  selectedKey: PropTypes.string.isRequired,
```

Comment:
Talked with @yuderekyu offline about this, but I think this prop would be better named 'initialMenuKey' or 'startingMenuKey' or something.

Reply:
+1 to `initialMenuKey`

Reply:
Refactored in [this](https://github.com/cerner/terra-framework/pull/103/commits/efd64c3eed5f7146aaefa0505339e34e92f75bed) commit (a lot of instances).